### DISCUSSION TITLE
Auto f/w updates?

### DISCUSSION THREAD

**Discussion by uxhamby** [arendst/Tasmota#19394] [Q&A]
Hi,

I have an 8266 that  had been running an older version of tasmota firmware (earlier than 13.1.0), I set this unit up a while ago, and have not interacted with it for 6 months or so. It was configured to show data on a 0.96" Oled display, so was running the (display) version of the f/w to support the display.

It appears as if this unit updated itself to 13.1.0 (tasmota), so the display no longer worked.

Do these update themselves if left running but unattended? Is there documentation on that somewhere?

I manually went in and installed 13.1.0 (displays) version of the firmware to get the display running again. All is good there.

Assuming they do update themselves in this manner, how can I set it up to select the right version, to update to, so as to continue supporting the hardware?

Thanks,

Brian H.
Uxbridge Ont.

**Comment by barbudor** [comment]
There is no autoupdate in tasmota
Once must either click on the Update button in the update firmware page or send a MQTT "Upgrade 1" command
Hardly happens by mistake

Are you using any kind of management software ?
TDM ?
TasmoAdmin ?
HomeAssistant ?

**Comment by uxhamby** [comment]
"Are you using any kind of management software ?
TDM ?
TasmoAdmin ?
HomeAssistant ?"

No, I am not using anything like that. Its just stand alone Tasmota here.

I have several units, two with (display) version and one with (sensor) version of the f/w, that have apparently auto updated to 13.0.0 (tasmota).

Thanks,

Brian H.

**Comment by arendst** [comment]
Tasmota does indeed NOT autoupdate .... until you told it to do so.

To be able to provide a hint pls provide output of commands ``status 0`` and ``rule0``.
